Rakefile

```diff
@@ -11,7 +11,7 @@ task :console do
   if File.exist? config_file
     config = YAML.load File.read(config_file)
 
-    AWS.config(config['aws'])
+    Aws.config = config['aws']
```

Comment:
I'm not running Shoryuken with Rails, so maybe I'm missing something. But I agree that initializers seem to be the way to go. Does that work for you @elsurudo?

@leemhenson your note is because of https://github.com/phstc/shoryuken/blob/master/Rakefile#L14? So now it's not going to work without removing the standard keys: https://github.com/musicglue/shoryuken/blob/aws-sdk-core/lib/shoryuken/cli.rb#L275-L279? Do you guys use this task?